### ISSUE TITLE
pointer on struct inside struct

### ISSUE THREAD

**Issue by Konstantin8105** [elliotchance/c2go#466]
Check after #460:
```c
typedef struct sqlite3_vfs sqlite3_vfs;
struct sqlite3_vfs {
  sqlite3_vfs *pNext;      /* Next registered VFS */
}
```

**Comment by elliotchance** [comment]
@Konstantin8105 Is this fixed now that #460 is merged?

**Comment by Konstantin8105** [comment]
yes
